vba_timeseries/models/value2pl/suuvid_obs_value2pl.m

```diff
@@ -0,0 +1,44 @@
+function  [ gx ] = suuvid_obs_nonu(Xt, phi, u, inG)
+% INPUT
+% - x_t : hidden states (weights of basis functions)
+% - phi : temperature (1x1)
+% - u   : imput vector (not used in observation)
+% - inG : multinomial
+% OUTPUT
+% - gx : p(chosen|x_t)
+
+phi = transform_phi(phi, inG);
+beta = phi(1); %motor speed recovery rate
```

Comment:
Overall, this file looks like the nonu version (no sign of nu), while the nonu one above (in the nonu subfolder) has been modified to be value2pl (with nu). So, you've swapped which code should be which.